lib/project-config.ts

```diff
@@ -8,49 +8,70 @@ export type projectInfo = {
     urlPrefix: string; // url to 'listserv' of mail (should it be in mailrepo?)
 };
 
+export interface IRepoConfig {
+    name: string; // name of the repo
+    owner: string; // owner of repo holding the notes (tracking data)
+    baseOwner: string; // owner of upstream ("base") repo
+    testOwner?: string; // owner of the test repo (if any)
+    owners: string[]; // owners of clones being monitored (PR checking)
```

Comment:
@webstech what do you think about the idea to move this attribute to `IAppConfig` and to rename it to `installedOn`? That way, the `repo` still can have an `upstreamOwner` on whose repository, however, GitGitGadget is _not_ installed...

Reply:
This is used several times in `CIHelper` so the location could be justified.  On the other hand, move it if you like.